Makefile.include

```diff
@@ -498,6 +501,7 @@ viewconf:
 	@echo "----------------- Make variables: --------------"
 	@echo "##### \"TARGET\": ________________________________ $(TARGET)"
 	@echo "##### \"BOARD\": _________________________________ $(BOARD)"
+	@echo "##### \"BOARD_REVISION\": _________________________$(BOARD_REVISION)"
```

Comment:
I don't know anything about these parts of the build system but I think there's a space missing between the underscores and the board revision.